.github/workflows/ci-pr.yaml

```diff
@@ -70,7 +71,7 @@ jobs:
           echo "testing $SERVICE..."
           # save current working dir to memory and cd to src/$SERVICE
           pushd src/$SERVICE
-            python3 -m venv $HOME/venv-$SERVICE
+            python3 -m venv --clear $HOME/venv-$SERVICE
```

Comment:
created issue for this, i think best to keep in separate pr's

Reply:
removed!